internal/store/postgres/migrations/000058_add_unique_job_upstream.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE JOB_UPSTREAM ADD CONSTRAINT unique_constraint_job_upstream UNIQUE (PROJECT_NAME, JOB_NAME, UPSTREAM_PROJECT_NAME, UPSTREAM_JOB_NAME, UPSTREAM_RESOURCE_URN);
```

Comment:
I know this is an unexpected situation , but what if 2 upstreams have the same `RESOURCE_URN` (shared destination table case)

should we enforce the `UPSTREAM_RESOURCE_URN` in job_upstream table ?

Reply:
2 upstream have same resource_urn means `upstream_job_name` are different, but `upstream_resource_urn` are same. it will be treated as different records

Reply:
ohh sorry, my bad